deploy/kubernetes/dolphinscheduler/Chart.yaml

```diff
@@ -35,11 +35,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 3.1.0
+version: 3.2.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 3.1.0
+appVersion: 3.2.2
```

Comment:
I don't think this `appVersion` should be changed.
 `dolphinscheduler.master.labels` quotes `appVersion`
https://github.com/apache/dolphinscheduler/blob/56ecec779ca7877c1cad9ab27fcd3a7055b8e74a/deploy/kubernetes/dolphinscheduler/templates/_helpers.tpl#L51-L64

And `dolphinscheduler.master.labels` as master's matchLabels, but matchLabels are immutable, which means that, Updating helm charts for an already deployed dolphinscheduler will fail
https://github.com/apache/dolphinscheduler/blob/56ecec779ca7877c1cad9ab27fcd3a7055b8e74a/deploy/kubernetes/dolphinscheduler/templates/statefulset-dolphinscheduler-master.yaml#L27-L29

Reply:
I have created a PR https://github.com/apache/dolphinscheduler/pull/16066 to fix this issue.
After merging, we can modify the appversion as normal.

@wangxj3 Please remember to cherry pick this PR https://github.com/apache/dolphinscheduler/pull/16066 before 3.2.2 is released